src/vector/backends/awkward_constructors.py

```diff
@@ -294,7 +314,7 @@ def Array(*args: typing.Any, **kwargs: typing.Any) -> typing.Any:
 
     if not _is_type_safe(array_type):
         raise TypeError("a coordinate must be of the type int or float")
-    fields = awkward.fields(akarray)
+    fields = awkward.fields(akarray).copy()
```

Comment:
`.copy()` can be removed.
```suggestion
    fields = awkward.fields(akarray)
```